### PR TITLE
feat: suggest chain

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/ui-kit/issues/32

### CODE REVIEW

File: packages/web-components/src/wallet-connection/suggestChain.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * @param {string} networkConfigHref
+ * @param {string=} caption
+ */
+export async function suggestChain(networkConfigHref, caption) {
+  // @ts-expect-error Check for Keplr
+  const { keplr } = window;
+
+  if (!keplr) {
+    throw Error('Missing Keplr');
+  }
```
Comment (turadg) [inline packages/web-components/src/wallet-connection/suggestChain.js:64]: please drop the suppression.
```suggestion
if (!('keplr' in window)) {
  throw Error('Missing Keplr');
}

const { keplr } = window;
```

also consider a global declaration that adds `keplr?: Keplr` to window so it's typed.

File: packages/web-components/src/wallet-connection/suggestChain.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  const networkConfig = await res.json();
+
+  if (!caption) {
+    const subdomain = url.hostname.split('.')[0];
+    caption = `Agoric ${subdomain}`;
+  }
+
+  const walletUrlForStaking = `https://${url.hostname}.staking.agoric.app`;
```
Comment (turadg) [inline packages/web-components/src/wallet-connection/suggestChain.js:80]: this template is more of a config. please group with other such config.

**Review by turadg — APPROVED** [review]
Approved contingent on a couple requests

**Comment by samsiegart** [comment]
Sorry, I enabled automerge on this mistakenly thinking it was my other pull request. I'm going to follow up right now with suggested changed.

**Pull request merged**